client/locations-list-map.js

```diff
@@ -79,31 +89,33 @@ const translatedFilterItems = (filterItems) => {
   }
 
   return translated;
-};
+}
 
 // get list of possible values for `Accepted Items`
 // iterates through data to extract all unique "accepting" items
 // matches against whitelist FILTER_ITEMS (from formEnumLookups.js)
 // and returns the i18n keys from FILTER_ITEMS for accepting items that match
 //
 // NOTE: the incoming data structure is very brittle; if that changes at all, this will break
-const parseFiltersFromData = (data) => {
+function parseFiltersFromData(data) {
   const acceptedItems = {};
   const orgTypes = {};
 
   Object.keys(data).forEach((state) => {
     Object.keys(data[state].cities).forEach((city) => {
       data[state].cities[city].entries.forEach((entry) => {
         // split on commas except if comma is in parentheses
-        entry.accepting.split(/, (?![^(]*\))/).map((a) => a.trim()).forEach((i) => {
-          const filterKey = i.toLowerCase();
-          if (FILTER_ITEMS[filterKey] !== undefined && acceptedItems[filterKey] === undefined) {
-            acceptedItems[filterKey] = {
-              ...FILTER_ITEMS[filterKey],
-              value: filterKey,
-            };
-          }
-        });
+        if (entry.accepting) {
```

Comment:
do we need filters on makers? I'm sure you thought of this, but it'd be easy to add a conditional in here on `dataset` to generate filters from different columns. We could even define dataset properties somewhere (`datasets: { ppe: { filters: ['org_type', 'accepting'] }, makers: { filters: ['capabilities'] } }`) and use that throughout to generalize this code more.

Reply:
I spent 2 hours last night hacking through filters for makers and gave up.

The answer is "yes", but I couldn't make it work so just wanted the list out first. :-/